contrib/babelfishpg_tsql/src/format.c

```diff
@@ -98,7 +97,7 @@ format_datetime(PG_FUNCTION_ARGS)
 	}
 	else
 	{
-		fmt_res = process_format_pattern(buf, format_pattern, data_type);
+		fmt_res = process_format_pattern(buf, (char *) format_pattern, (char *) data_type);
```

Comment:
Same as above, should be able to change format_pattern and data_type to const char *